src/md/src/QualityGates/new-errors-rule.md

```diff
@@ -0,0 +1,87 @@
+## Quality Gates: “New Errors” rule
```

Comment:
I suppose that it will be the part of `Quality Gates` section in docs.
So may be we can leave just `"New Errors" rule`?